### PR TITLE
feat(locale/pl): Add Polish language translations

### PR DESCRIPTION
**Pull request by requestrip**
**Pull Request Description**

_Please include a general description of the changes made, why they were necessary, 
and any other general things to note._

**Pull Request Checklist**:
* [x] Have you followed the guidelines in our Contributing document and Code of Conduct?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/project-error/npwd/pulls) for the same update/change?
* [x] Have you built and tested NPWD in-game after the relevant change?

### CODE REVIEW

**Comment by TasoOneAsia** [comment]
Hi @requestrip, thank you for the PR!

It appears that your PR has failed our CI test as your JSON is invalid. Could you please fix that before we merge this :)

File: phone/src/locale/pl.json [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+    "GENERIC_WAIT": "Proszę czekać...",
+    "GENERIC_SAVE": "Zapisz",
+    "GENERIC_EDIT": "Edytuj",
+    "GENERIC_UPDATE": "Aktualizacja",
+    "GENERIC_REQUIRED": "Wymagany",
+    "GENERIC_DELETE": "Usunać",
+    "GENERIC_REPORT": "Zgłoszenie",
+    "GENERIC_CALL": "Połączenie",
+    "GENERIC_MESSAGE": "Wiadomość",
+    "GENERIC_CLOSE": "Zamknij,
```
Comment (RockySouthpaw) [inline phone/src/locale/pl.json:282]:
```suggestion
    "GENERIC_CLOSE": "Zamknij",
```
Missing quotation here it looks like

**Comment by requestrip** [comment]
Oh i missed one " thank you @RockySouthpaw  for finding it. Now should work

**Comment by TasoOneAsia** [comment]
I'll sort this out for you, no worries.

**Pull request merged**